### ISSUE TITLE
slides 2d note with keyboard not just drag

### ISSUE THREAD

**Issue by hchiam** [hchiam/slides#125]
moving text with keyboard only should have audio feedback too

(also separate issue for moving image with keyboard for sighted keyboard-only users)
